src/components/EventCard.js

```diff
@@ -63,6 +63,25 @@ export default function EventCard(props) {
     setOpen(false);
   };
 
+  // Image Verification Stufff, better ideas than this, come on lesgoo !!
+
+  const doesImageExist = (url) =>
```

Comment:
Good use of a promise to resolve. 
Any reason you preferred resolving with a boolean, instead of just either resolving or rejecting?

Reply:
I wrote this as I use to write a function, not and not validate some
Request for server, we only have 2 outputs (True/False)


`also special note, we have done nothing wrong, it's the DB's mistake for going down xD so no need to omit error`

Jokes aside, I usually resolve when there are less no of outputs.

Reply:
Lmao yep the DB shouldnt be going down;
It doesnt look like this function can reject during most working conditions, so looks good to me